templates/cloud/index.html

```diff
@@ -256,6 +256,37 @@ <h2 class="muted-heading">Leading vendors publish on Juju</h2>
     </div>
 </section>
 
+<section class="row strip-dark">
+    <div class="strip-inner-wrapper">
+        <div class="eight-col">
+            <h2>Landscape</h2>
+            <h3>Server management</h3>
+            <p>
+                Landscape allows you to manage thousands of Ubuntu machines as
+                easily as one, making the administration of Ubuntu desktops,
+                servers and cloud instances more cost-effective. It&rsquo;s
+                easy to set up, easy to use and it requires no special
+                hardware. It features:
+            </p>
+            <ul class="four-col list-ticks--compact">
```

Comment:
There’s a gap below the first list on small, wrap with a div with the combined-list class 

![screen shot 2017-04-05 at 13 06 12](https://cloud.githubusercontent.com/assets/2152/24704713/b568ec9c-1a00-11e7-867e-4d57409c9a08.png)